deploy/olm-catalog/performance-addon-operator/0.0.1/performance-addon-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -35,6 +35,7 @@ metadata:
         }
       ]
     capabilities: Basic Install
+    containerImage: REPLACE_IMAGE
```

Comment:
weird that this got added. happened to me locally as well when I tried it. There's supposed to be a ci check to ensure we can't merge unless this csv is synced. 

i'll look into it. it's not related to this pr